.idea/misc.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/Denisa/Desktop/Summer Practice/summerPractice-2022/app/src/main/res/drawable-v24/ic_launcher_foreground.xml" value="0.1" />
+        <entry key="..\:/Users/Denisa/Desktop/Summer Practice/summerPractice-2022/app/src/main/res/drawable/ic_launcher_background.xml" value="0.1" />
+        <entry key="..\:/Users/Denisa/Desktop/Summer Practice/summerPractice-2022/app/src/main/res/drawable/splash_background.xml" value="0.1" />
+        <entry key="..\:/Users/Denisa/Desktop/Summer Practice/summerPractice-2022/app/src/main/res/layout/activity_main.xml" value="0.10733695652173914" />
+      </map>
+    </option>
+  </component>
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
```

Comment:
you shouldn't commit this file, it contains your PC setup with your name, please find a way to uncommit it